invenio_rdm_records/resources/serializers/datacite/schema.py

```diff
@@ -571,6 +571,21 @@ def get_rights(self, obj):
                 entry["rightsUri"] = link
             serialized_rights.append(entry)
 
+        # Adding access_right information
+        access_right = obj["access"]["status"]
+        if access_right == "metadata-only":
+            access_right = "closed"
+
+        access_right_formatted = access_right.capitalize() + " Access"
+        rights_uri = "info:eu-repo/semantics/{}Access".format(access_right)
```

Comment:
Other string formatting in this file are using f-strings; should we use an f-string here too for consistency?